### ISSUE TITLE
weird characters appear when typing in new symbol and list atoms

### ISSUE THREAD

**Issue by Spacechild1** [pure-data/pure-data#1422]
when typing things in the new symbol and list atoms, a random characters starts to appear right before the `...`.

Examples:

![symbol1](https://user-images.githubusercontent.com/16126632/139531927-3f818406-c021-450b-a51f-a5232a8c33c5.JPG)

![symbol2](https://user-images.githubusercontent.com/16126632/139531932-bdc2dfa1-5973-4603-ad2b-8c541925c960.JPG)

Pd 8897538f883c4bec907647989c1198a9ef9495d7 on Windows 7

**Comment by Lucarda** [comment]
Confirmed

Pd(amd64) 8897538 on Windows 10

**Comment by Ant1r** [comment]
this doesn't seem to happen on Linux (neither Metacity nor gnome-shell).

**Comment by umlaeute** [comment]
an i can't reproduce it in a VM either.

however, running Pd (`develop` 6892cc3f519f6aefe979303dd910868d6e63ee63  on Linux) through Valgrind indicates that there's a memory problem:

~~~
==1068044== Invalid read of size 1
==1068044==    at 0x1D2E51: u8_nextchar (s_utf8.c:246)
==1068044==    by 0x18AB06: rtext_formatatom (g_rtext.c:323)
==1068044==    by 0x18AB06: rtext_senditup (g_rtext.c:393)
==1068044==    by 0x18C099: rtext_key (g_rtext.c:716)
==1068044==    by 0x1A49F9: gatom_key (g_text.c:792)
==1068044==    by 0x1BCAA7: binbuf_eval (m_binbuf.c:790)
==1068044==    by 0x1C8B6D: socketreceiver_read (s_inter.c:658)
==1068044==    by 0x1C8B6D: socketreceiver_read (s_inter.c:590)
==1068044==    by 0x1C7B33: sys_domicrosleep (s_inter.c:226)
==1068044==    by 0x1C9AF0: sys_pollgui (s_inter.c:976)
==1068044==    by 0x1C495B: sched_idletask (m_sched.c:299)
==1068044==    by 0x1C54A6: m_pollingscheduler (m_sched.c:375)
==1068044==    by 0x1C54A6: m_mainloop (m_sched.c:432)
==1068044==    by 0x4BD3E49: (below main) (libc-start.c:314)
==1068044==  Address 0x53bc1d1 is 0 bytes after a block of size 1 alloc'd
==1068044==    at 0x483E82F: realloc (vg_replace_malloc.c:1437)
==1068044==    by 0x1C1733: resizebytes (m_memory.c:55)
==1068044==    by 0x18C2D9: rtext_key (g_rtext.c:637)
==1068044==    by 0x1A49F9: gatom_key (g_text.c:792)
==1068044==    by 0x1BCAA7: binbuf_eval (m_binbuf.c:790)
==1068044==    by 0x1C8B6D: socketreceiver_read (s_inter.c:658)
==1068044==    by 0x1C8B6D: socketreceiver_read (s_inter.c:590)
==1068044==    by 0x1C7B33: sys_domicrosleep (s_inter.c:226)
==1068044==    by 0x1C9AF0: sys_pollgui (s_inter.c:976)
==1068044==    by 0x1C495B: sched_idletask (m_sched.c:299)
==1068044==    by 0x1C54A6: m_pollingscheduler (m_sched.c:375)
==1068044==    by 0x1C54A6: m_mainloop (m_sched.c:432)
==1068044==    by 0x4BD3E49: (below main) (libc-start.c:314)
==1068044== 
~~~

**Comment by Spacechild1** [comment]
closed via #1456
